packages/ui/src/components/pagination/acvPagination.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts" setup>
+  import { computed } from 'vue';
+  import AcvPaginationItem from './acvPaginationItem.vue';
+  import type { AcvPaginationProps } from '@/components';
+
+  const props = defineProps<AcvPaginationProps>();
+
+  const maximumVisibleSlotsInPagination = 7;
+  const page = computed(() => props.pagination.page.value);
+  const lastPage = computed(() => props.pagination.lastPage.value);
+</script>
+
+<template>
+  <div class="acv-pagination">
+    <!-- Previous page button -->
+    <AcvPaginationItem
+      :page="page - 1"
+      :disabled="page <= 1"
+    >
+      &#8592;
+    </AcvPaginationItem>
+    <!-- First page -->
+    <AcvPaginationItem
+      :page="1"
+      :active="page === 1"
+    />
+    <!-- Pages 2-5 or ellipsis -->
+    <template v-if="lastPage <= maximumVisibleSlotsInPagination">
+      <template v-for="p of 5">
```

Comment:
We should discuss number of displayed pages. Now we have some cases with more than 10 displayed pages